uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -69,18 +69,24 @@ static KEYWORDS: Lazy<HashSet<String>> = Lazy::new(|| {
     HashSet::from_iter(kwlist.into_iter().map(|s| s.to_string()))
 });
 
+fn default_false() -> bool {
+    false
+}
+
 // Config options to customize the generated python.
 #[derive(Debug, Clone, Default, Serialize, Deserialize)]
 pub struct Config {
     cdylib_name: Option<String>,
     #[serde(default)]
     custom_types: HashMap<String, CustomTypeConfig>,
+    #[serde(default = "default_false")]
+    json_support: bool,
 }
 
 #[derive(Debug, Clone, Default, Serialize, Deserialize)]
 pub struct CustomTypeConfig {
-    // This `CustomTypeConfig` doesn't have a `type_name` like the others -- which is why we have
-    // separate structs rather than a shared one.
+    into_type: Option<String>,
+    from_type: Option<String>,
```

Comment:
Please ignore these, they're from a previous attempt at this and I forgot to remove them 😬